### ISSUE TITLE
Return an array from "loop()" instead of an object

### ISSUE THREAD

**Issue by jgoux** [raisemarketplace/redux-loop#27]
Hi,

is there any reason why the `loop` function returns an object ?

As my experiment (which is turning into a real app now :smile: ) grow, I end up writing this kind of code **a lot** : 
```js
const update = (model = initialModel, action) => {
  switch(action.type) {
    case MODIFY_THING:
      const { model: m, effect: e } = thing.update(model.thing, action.payload.action)
      return loop(
        { ...model, thing: m},
        Effects.batch([
          Effects.lift(e, modifyThing)
        ])
      )
    default:
      return loop(
        model,
        Effects.none()
      )
}
```

It's a tiny detail, but being able to directly extract the `model` and the `effect` from the `loop` function would be a great DX boost : 
```js
const [m, e] = thing.update(/*...*/)
```

Also it would be closer to Elm's API, which uses a Tuple representation.

**Comment by lukewestby** [comment]
Included in v2.0.0!

**Comment by jgoux** [comment]
:100: :+1: :star2: :sparkler: :smile_cat:
